tests/test_util.py

```diff
@@ -101,102 +100,66 @@ def get_random_hamiltonian_and_inference(qubits,
   return random_qhbm.modular_hamiltonian, random_qhbm
 
 
-def get_random_pauli_sum(qubits):
-  """Test fixture.
-    Args:
-      qubits: A list of `cirq.GridQubit`s on which to build the pauli sum.
-    Returns:
-      pauli_sum: A `cirq.PauliSum` which is a linear combination of random pauli
-      strings on `qubits`.
-    """
-  paulis = [cirq.X, cirq.Y, cirq.Z]
+def random_hermitian_matrix(num_qubits):
+  """Returns a random Hermitian matrix.
 
-  coeff_max = 1.5
-  coeff_min = -1.0 * coeff_max
+  Uses the property that A + A* is Hermitian for any matrix A.
 
-  num_qubits = len(qubits)
-  num_pauli_terms = max(1, num_qubits - 1)
-  num_pauli_factors = max(1, num_qubits - 1)
+  Args:
+    num_qubits: Number of qubits on which the matrix acts.
+  """
+  dim = 2**num_qubits
+  val_range = 2
+  random_real = tf.cast(
```

Comment:
why is this real and the below is imaginary? seems they are the same call?

Reply:
Good catch, should be multiplied by 1j